components/steering_wheel/include/IO_Digital.h

```diff
@@ -0,0 +1,58 @@
+/**
+ *IO_Digital.h
+ *Digital IO Module Header
+ */
+
+#pragma once
+
+/************************************************
+ * Includes
+ * *********************************************/
+
+/************************************************
+ * Defines
+ * *********************************************/
+
+#define NUM_INPUTS 7
+
+/************************************************
+ * Typedefs
+ * *********************************************/
+
+enum INPUT
```

Comment:
don't do this, the only reason you'd really do this is if you're iterating over them